### PR TITLE
add documentation of appt avail table injection

### PR DESCRIPTION
**Pull request by ChemiKyle**
Adds documentation for appointment table injection introduced in PR #19 

Adds a new image for an example appointment table and updates the project level config image. I also corrected a typo or 2 somewhere.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,32 @@ The module will define a cron job that runs daily to assure _appointment\_horizo
 
 As of release 0.6.0, the most precise way to make appointment blocks is to set _appointment\_horizon_ days to zero and set _start\_date_ to the precise date for which you want to make appointments. Select the sites that need new appointment blocks and click `Manually generate new appointments`. Repeat this process for each site date that needs new appointment blocks.
 
+## Showing Appointment Availability
+
+A table showing the current number of available appointments can be shown on any survey page after a few steps. Some SQL knowledge is required.  
+![Appointment availabilty table example](images/appointment_table.png)
+1. Create a query with [REDCap Webservices](https://github.com/ctsit/redcap_webservices) to deliver the data you wish to display.  
+    example:
+    ```sql
```
Comment (pbchase) [inline README.md]: A little extra white space improves the formatting in MacDown and should not impede to the formatting in Github

```suggestion
![Appointment availabilty table example](images/appointment_table.png)

1. Create a query with [REDCap Webservices](https://github.com/ctsit/redcap_webservices) to deliver the data you wish to display.  
    example:

    ```sql
```

File: README.md [hunk 2]
```diff
@@ -69,6 +69,32 @@ The module will define a cron job that runs daily to assure _appointment\_horizo
...
+    CONCAT(site_long_name, ' (', site_short_name, ')') AS location,
+        ts.testing_type,
+        COUNT(IF(fra.record_id IS NULL, 1, NULL)) AS available
+            FROM redcap_entity_fr_appointment AS fra
+            INNER join redcap_entity_test_site AS ts ON (fra.site = ts.id)
+            WHERE ts.project_id = [project-id]
+            AND ( fra.appointment_block > UNIX_TIMESTAMP( DATE( NOW() + INTERVAL IF(HOUR(NOW()) >= 16, 2, 1) DAY ) ) )
+            GROUP BY site
+    ```
```
Comment (pbchase) [inline README.md:90]: A little extra white space improves the formatting in MacDown and should not impede to the formatting in Github.

```suggestion
    ```

```

File: README.md [hunk 3]
```diff
@@ -28,8 +28,8 @@ To configure and use this module, follow these steps:
...
-1. Adjust the project_ids referenced on those just-loaded sites by editing and running a copy of [`redcap_entity_test_site_update.sql`](example/redcap_entity_test_site_update.sql)
+1. Use a MySQL client to load sites data from [`redcap_entity_test_site_data.sql`](example/redcap_entity_test_site_data.sql). Alternatively, use the `Define Sites` page of this module to enter the data. Be cautious as some portions of the interface of this module have very few guard rails. I.e., your data entry will not be checked. E.g., items must be entered in a 4-digit, 24-hour clock format such that 7 a.m. is 0700. The leading zero is critical. The development schedule has not allowed the addition of these tests, nor does it allow for much documentation. As such, the authors advise you to _use the example configuration_ as a starting point.
```
Comment (pbchase) [inline README.md:32]:
```suggestion
1. The field _REDCap Webservices url for appointment data_ is an optional feature explained in [Showing Appointment Availability](#showing-appointment-availability)
1. Use a MySQL client to load sites data from [`redcap_entity_test_site_data.sql`](example/redcap_entity_test_site_data.sql). Alternatively, use the `Define Sites` page of this module to enter the data. Be cautious as some portions of the interface of this module have very few guard rails. I.e., your data entry will not be checked. E.g., items must be entered in a 4-digit, 24-hour clock format such that 7 a.m. is 0700. The leading zero is critical. The development schedule has not allowed the addition of these tests, nor does it allow for much documentation. As such, the authors advise you to _use the example configuration_ as a starting point.
```

Explain where to get info on the new parameter.

**Comment by pbchase** [comment]
@ChemiKyle, this looks good. Thanks for adding the docs.

File: README.md (hunk 3 above)
Comment (ChemiKyle) [reply README.md:32]: I'm glad you caught this, I meant to do this.

**Comment by ChemiKyle** [comment]
@pbchase, all suggestions have been tested and committed. Merge if you're happy

___

A quick note on the whitespace commits:  
I've never used MacDown, but I've noticed differences with other renderers and the final result on GitHub, _especially wrt ordered lists and whitespace_. I use [grip](https://github.com/joeyespo/grip) to preview rendered markdown, it uses the GitHub API for rendering; I'd highly recommend using it at least as a last step - it auto-refreshes when the text of the `.md` file is changed so it's also well suited to live editing if you want to replace MacDown with [a better editor](https://www.gnu.org/fun/jokes/ed-msg.en.html).

**Comment by pbchase** [comment]
@ChemiKyle, thanks for the tip on grip. It bothered me a bit that I am using macdown as my standard. It is good but not great.

**Pull request merged**